### PR TITLE
doc: Add section about MDS cache limits

### PR DESCRIPTION
**Pull request by baruza**
Signed-off-by: Bara Ancincova (bara@redhat.com)

### CODE REVIEW

**Comment by baruza** [comment]
ah, closed by mistake.. reopening

File: doc/cephfs/cache-size-limits.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Understanding MDS Cache Size Limits
+===================================
+
+This section describes ways to limit MDS cache size.
+
+You can limit the size of the Metadata Server (MDS) cache by:
+
+* *A memory limit*: A new behavior introduced in the {product} 3. Use the `mds_cache_memory_limit` parameters. Red Hat recommends to use memory limits instead of inode count limits.
```
Comment (batrick) [inline doc/cephfs/cache-size-limits.rst]: In upstream docs, we refer to Ceph by its release name (Luminous in this case).

Comment (batrick) [reply doc/cephfs/cache-size-limits.rst]: Also, use "We recommend" instead of "Red Hat".

**Comment by batrick** [comment]
jenkins render docs

**Comment by batrick** [comment]
http://docs.ceph.com/ceph-prs/19212/cephfs/cache-size-limits/

Looks good to me.

File: doc/cephfs/cache-size-limits.rst (hunk 1 above)
Comment (baruza) [reply doc/cephfs/cache-size-limits.rst]: Ah, you are completely right, I've missed that.. The new version should be ok :)

**Pull request merged**